docs/articles/documentation/reference/test-api/testcontroller/eval.md

```diff
@@ -27,7 +27,9 @@ test('My Test', async t => {
 });
 ```
 
-> Since the `eval` method returns a value, and not an object, you cannot chain other methods of the test controller after 'eval'.
+> Important! Since `eval` returns a promise-wrapped value instead of a local context, you cannot chain other methods of the test controller after `eval`.
+>
+> Always place `eval` in a separate call.
```

Comment:
It would be nice to add the bad variant and the good one

Reply:
added